htmresearch/regions/PoolingRegion.py

```diff
@@ -243,7 +249,14 @@ def compute(self, inputs, outputs):
     The guts of the compute are contained in the self._poolerClass compute() call
     """
     activeCells = inputs["activeCells"]
-    predictedActiveCells = inputs["predictedActiveCells"]
+    predictedActiveCells = inputs["predictedActiveCells"] if (
```

Comment:
If this is a required input, should we throw an exception if not connected? If so, the exception can be thrown in initialize, which is always called once before the first all to compute().